catalog-info.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: analysis-platform
+  title: Analysis Platform
+  description: >
+    The data analysis workflows platform deployment. The deployment consists of a Kubernetes virtual cluster (vcluster),
+    in which Argo Workflows is deployed alongside accompanying services and a monitoring stack. Policies are created with
+    Kyverno and continuous delivery is achieved with ArgoCD.
+   - argocd
+  tags:
+    - kubernetes
+    - argo-workflows
+    - argo-cd
+    - kyverno
+    - vcluster
+    - monitoring
+spec:
+  owner: group:data-analysis
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: argo-workflows-openapi
+  title: Argo Workflows OpenAPI
+  description: Simple service to expose the Argo Workflows Schema
```

Comment:
Maybe I don't understand, this package isn't a service? It's just a library that converts the Argo Workflows API to Rust types.

This isn't a library that we expect (or want) anyone else to use. I wouldn't list it in the catalog.